Packs/PhishLabs/Scripts/PhishLabsPopulateIndicators/PhishLabsPopulateIndicators.py

```diff
@@ -18,79 +18,88 @@ def indicator_type_finder(indicator_data: dict):
     # "ReturnPath" and so on - to combat that we find the Email indicator type by regex
     # returned URLs could fit the email regex at some cases so we exclude them
     if re.match(str(emailRegex), str(indicator)) and str(indicator_data.get('type')).lower() != 'url':
-        return 'Email'
+        return FeedIndicatorType.Email
+
+    elif (re.match(sha256Regex, indicator) or re.match(md5Regex, indicator)  # type: ignore
+            or re.match(sha1Regex, indicator)):  # type: ignore
+        return FeedIndicatorType.File
```

Comment:
I think there are also `FeedIndicatorType.MD5`, `FeedIndicatorType.SHA1` etc. Not sure if it matters.

Reply:
@ronykoz @David-BMS - I believe this should be only file right?

Reply:
We are only using the File type yes